### PR TITLE
Fix #141: Item renames

### LINKED ISSUE #141: Item renames
Renames:

'runInfo' -> 'run'
'toolInfo' -> 'tool'
'ruleInfo' -> 'rules'
'invocationInfo' -> 'invocation'

### PR DESCRIPTION
**Pull request by ghost**
@michaelcfanning 

`toolInfo` => `tool`
`runInfo` => `run`
`invocationInfo` => `invocation`
`fileInfo` => `files`
`fileReference` => `file`
`complianceInfo` => `compliance` (even though it's not included in spec at present)
`runStartTime` => `startTime`
`runEndTime` => `endTime`
`ruleDescriptor` => `rule`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,12 +5,6 @@ This repo contains an early draft of the specification document for the Static A
 
 Please read the LICENSE file in the repo root directory, which explains the conditions for participating in the review of this draft specification.
 
-## Version
-
-The spec currently specifies that the `version` property of the top-level `issueLog` object shall be `0.1`.
-We will increment this gradually as the spec evolves, and the first version of the resulting standard will
-specify `1.0`.
-
```
Comment (ghost) [inline README.md:13]: Unrelated change. It wasn't true, wasn't worth keeping up to date, and generally didn't make the world a better place.

File: document/CoreFormat.mdk [hunk 2]
```diff
@@ -303,40 +303,40 @@ Suppose a tool is invoked with the command line
...
+Viewers will be able to provide the most information to users if the `files`
 property is present and contains information for every file in which results were detected.
 ~
 
-Each property name in the `fileInfo` object shall be the URI of a scanned file.
-No two of these property names shall be equivalent as defined in &sect;[#uriProperties].
-
-Each property value in the `fileInfo` object shall be an array of `fileReference` objects (&sect;[#fileReference])
-which contain information about the file identified by the URI in the property name.
-
```
Comment (ghost) [inline document/CoreFormat.mdk:339]: I moved these two paragraphs up above the IsoNumberedNote, because the note referred to the `fileReference` (now `file`) object, which hadn't yet been mentioned.

File: document/PolicyComplianceProfile.mdk [hunk 3]
```diff
@@ -15,28 +15,28 @@ The team must also be able to verify that the tool actually ran the necessary ru
...
 tool producing the log was configured in a way that complies with applicable policies.
 
-A `complianceInfo` object is not a policy or statement of policy.
+A `compliance` object is not a policy or statement of policy.
 It is merely a collection of information that helps determine policy compliance.
 
-### `enabledRules` property { #complianceInfo-enabledRules }
+### `enabledRules` property { #compliance-enabledRules }
 
-A `complianceInfo` object (&sect;[#complianceInfo])
```
Comment (ghost) [inline document/PolicyComplianceProfile.mdk:39]: Removed the section reference because nowhere else in the spec do I explicitly reference the object from the description of one of its properties.

File: document/CoreFormat.mdk [hunk 4]
```diff
@@ -198,50 +198,50 @@ each of which represents a single result produced in the course of a run.
...
 ~ IsoExample
 `"CodeScanner 2.0, Developer Preview (en-US)"`
 ~
 
-### `version` property { #toolInfo-version }
+### `version` property { #tool-version }
 
-A `toolInfo` object should contain a property named `version` whose value is a string containing the tool version.
+A `tool` object should contain a property named `version` whose value is a string containing the tool version.
 If present, the string should be in the format specified by Semantic Versioning 2.0.0.
```
Comment (ghost) [inline document/CoreFormat.mdk]: I'd like to get rid of this passage where we seem to be telling tool vendors how to version their tools. I'd delete lines 245-253.

File: document/CoreFormat.mdk [hunk 5]
```diff
@@ -183,113 +179,22 @@ and contains the output of that run.
 ```
 ~
 
-### `toolInfo` property { #runLog-toolInfo }
```
Comment (ghost) [inline document/CoreFormat.mdk:186]: This information hasn't gone away. What's really happened is that the information from below that was previously under `run` (the former `runInfo`) has moved upwards in the doc to live directly under the new `run` (the former `runLog`). This will be easy for you to see in the HTML.

**Comment by michaelcfanning** [comment]
LGTM
